src/App.jsx

```diff
@@ -1,104 +1,24 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList/PostList';
```

Comment:
You have an `index.js` file in each folder, so you can just write `'./components/PostList'` (same for the others)